src/sentry/models/release.py

```diff
@@ -469,11 +469,13 @@ def set_commits(self, commit_list):
                 )
 
         commit_resolutions = list(
-            GroupCommitResolution.objects.filter(
-                commit_id__in=ReleaseCommit.objects.filter(release=self)
+            GroupLink.objects.filter(
+                linked_type=GroupLink.LinkedType.commit,
+                linked_id__in=ReleaseCommit.objects.filter(release=self)
                 .values_list('commit_id', flat=True),
-            ).values_list('group_id', 'commit_id')
+            ).extra(select={'commit_id': 'linked_id'}).values_list('group_id', 'commit_id')
```

Comment:
is this `extra` actually necessary? couldn't you just change it to `.values_list('group_id', 'linked_id')`